packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -253,15 +254,44 @@ export const validateCart = async (
   const {
     clients: { commerce },
     loaders: { skuLoader },
+    headers
   } = ctx
 
   // Step1: Get OrderForm from VTEX Commerce
   const orderForm = await getOrderForm(orderNumber, session, ctx)
 
+  
+  // Validate if Session's cookie exists
+  const cookieSession = getCookie('vtex_session', headers.cookie) 
+
+  if (cookieSession && enableOrderFormSync === true) {
+
+    const {namespaces} =  await commerce.getsessionorder()  
```

Comment:
If you agree with camelCase we need to replace here as well 😉 
```suggestion
    const { namespaces } = await commerce.getSessionOrder()  
```